parent-conf/pom.xml

```diff
@@ -64,7 +64,7 @@
         <spark.sql.kafka.version>2.4.2</spark.sql.kafka.version>
```

Comment:
I still think that `spark.sql.kafka.version` should be equal to `spark.version`, because they are released together

Reply:
Agree. Just updated.